R/load.R

```diff
@@ -82,18 +82,18 @@ find_load_strategy <- function(x, option = roxy_meta_get("load", "pkgload")) {
   if (is.null(x)) {
     x <- option
     if (!is.character(x) || length(x) != 1) {
-      abort("roxygen2 `load` option must be a string")
+      cli::cli_abort("roxygen2 `load` option must be a string")
```

Comment:
To we want to write 
```
      cli::cli_abort("roxygen2 {.code load} option must be a string")
```
here and also below?  At some point we should write up some conventions.

Reply:
Or `.arg`?